### PR TITLE
Implement todo application with react-hooks

### PR DESCRIPTION
**Pull request by VIanor**
https://app.asana.com/0/1114938562388642/1114943307439258

### CODE REVIEW

File: components/Todo.js [hunk 1]
```diff
@@ -8,77 +8,73 @@ import {
   CheckBox
 } from 'react-native'
 
-export default class Todo extends React.Component {
-  state = {
-    edit: false,
-    text: ''
-  }
+export default function Todo (props) {
```
Comment (zheoreh) [inline components/Todo.js]: @VIanor `Todo ({data, ...props})` and replace everywhere `props.data` with `data`

File: components/Todo.js [hunk 2]
```diff
@@ -8,77 +8,73 @@ import {
...
-          <CheckBox
-            onChange={() => {
-              this.props.toggleStatus(data.id)
-            }}
-            value={data.completed}
-            checked={data.completed}
-          />
-        </View>
-        {this.state.edit ? (
+  let { data } = props
```
Comment (zheoreh) [inline components/Todo.js]: @VIanor this must be removed because of a reason from previous comment.

File: components/Todos.js [hunk 3]
```diff
@@ -10,81 +10,80 @@ import {
...
-    text: '',
-    items: []
-  }
+export default function Todos () {
+  const [text, setText] = useState('')
+  const [items, setItems] = useState([])
+
+  const addItem = () => {
+    if (!text.trim()) return
+    const newItem = {
```
Comment (zheoreh) [inline components/Todos.js]: @VIanor just `item`

File: components/Todos.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
-import React from 'react'
-import TodoModel from './models/Todo'
+import React, { useState } from 'react'
+import idGenerator from 'react-id-generator'
```
Comment (zheoreh) [inline components/Todos.js]: @VIanor restore the model

File: components/Todo.js (hunk 1 above)
Comment (VIanor) [reply components/Todo.js]: how can I rename the local save function or save props for avoid the bug?

File: components/Todo.js (hunk 2 above)
Comment (VIanor) [reply components/Todo.js]: @zheoreh done.

File: components/Todos.js (hunk 3 above)
Comment (VIanor) [reply components/Todos.js]: @zheoreh done.

File: components/Todos.js (hunk 4 above)
Comment (VIanor) [reply components/Todos.js]: @zheoreh done.

File: components/Todo.js (hunk 1 above)
Comment (zheoreh) [reply components/Todo.js]: @VIanor gave an answer on skype

Comment (VIanor) [reply components/Todo.js]: @zheoreh done.

**Pull request merged**